src/config/schema.ts

```diff
@@ -219,6 +219,27 @@ export const configSchema = {
       type: "object",
       properties: {
         initialStatus: { type: "string", enum: Object.values(AccountStatus) },
+        customFields: {
+          type: "array",
+          items: {
+            type: "object",
+            properties: {
+              name: { type: "string" },
+              type: {
+                type: "string",
+                enum: ["string", "integer", "float", "boolean"],
+              },
+              defaultValue: { type: ["string", "number", "boolean", "array"] },
+              array: { type: "boolean", default: false },
+              required: { type: "boolean", default: false },
+              editable: { type: "boolean", default: false },
+              index: { type: "boolean", default: false },
+            },
+            required: ["name", "type"],
```

Comment:
actually type is required here. ie: https://github.com/GaloyMoney/galoy/pull/1465/files#r944314491

Reply:
Fixed https://github.com/GaloyMoney/galoy/commit/c77987b90c2efa222f03c8aa24b6e792af811ba2 also please check comment here https://github.com/GaloyMoney/galoy/pull/1465#discussion_r948646070